tests/util/diffraction_calibration_synthetic_data.py

```diff
@@ -10,6 +10,7 @@
 from collections import namedtuple
 from collections.abc import Sequence
 from pathlib import Path
+from re import S
```

Comment:
I don't think this import is being used.

Reply:
pylance being "helpful"